src/legacy/core_plugins/data/public/filter/filter_bar/filter_editor/lib/filter_editor_utils.ts

```diff
@@ -176,7 +176,10 @@ function buildBaseFilter(
     case 'phrases':
       return buildPhrasesFilter(field, params, indexPattern);
     case 'range':
-      const newParams = { gte: params.from, lt: params.to };
+      const newParams = {
+        gte: params.from,
+        lte: params.to,
+      };
```

Comment:
I don't think we should change the bounds of the filter created by the filter editor. Users may be used to the existing behavior and this could be a subtle change that trips them up.

Reply:
If you say a filter is between two values would you not expect those values are included? and not between to be excluded? Not a mix of equalities.

Reply:
Maybe, but I think it is somewhat common for ranges to have inclusive beginnings but exclusive ends. Javascript's `slice` for example. 

I don't remember for sure, but I think we chose to use `gte` and `lt` in the filter editor because that's how [histogram visualizations have created filters](https://github.com/elastic/kibana/blob/8837faebde0a6efce864d497f3dd62ba06bf396b/src/legacy/ui/public/agg_types/buckets/create_filter/histogram.ts#L25) since forever. There was longstanding precedent to follow there. The range slider is the newest of these three ways to create a range filter, so if anything it was breaking with convention by using `lte`. I generally try to stick with current behavior unless there's a strong reason to change it.